### ISSUE TITLE
native-image-maven-plugin rc11 on maven central is missing dependencies

### ISSUE THREAD

**Issue by pmlopes** [oracle/graal#921]
The maven plugin published to maven central refers to: `svm-hosted-native-windows-amd64` which is not be be found there, while it is possible to exclude this dependency when using:

```xml
    <dependency>
      <groupId>com.oracle.substratevm</groupId>
      <artifactId>svm</artifactId>
      <version>1.0.0-rc11</version>
      <scope>provided</scope>
      <exclusions>
        <exclusion>
          <groupId>com.oracle.substratevm</groupId>
          <artifactId>svm-hosted-native-windows-amd64</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
```

This isn't possible for a plugin which makes the plugin not work with the current latest version.

**Comment by olpaw** [comment]
@pmlopes thanks for reporting.

**Comment by flortsch** [comment]
The missing artifact has been released and should be available on Maven Central soon.

**Comment by pmlopes** [comment]
I can confirm that it works now! :+1:

**Comment by olpaw** [comment]
As a heads-up ... if you are running on Linux with binutils 2.31.1 (e.g. like Fedora 29 does) building with rc11
```
 <plugin>
  <groupId>com.oracle.substratevm</groupId>
  <artifactId>native-image-maven-plugin</artifactId>
  <version>1.0.0-rc11</version>
  <executions>
    <execution>
      <goals>
        <goal>native-image</goal>
      </goals>
      <phase>package</phase>
    </execution>
  </executions>
</plugin>
```
will fail during the linking step. That is caused by a bug in binutils (elfutils specifically). See
https://sourceware.org/bugzilla/show_bug.cgi?id=23919

The issue will go away once distributions update to binutils 2.32 (or backport the fix to 2.31).
